stock_move_line_change_lot/models/stock_move_line.py

```diff
@@ -161,6 +165,25 @@ def write(self, vals):
         res &= super(StockMoveLine, self - already_processed).write(vals)
 
         if to_reassign_moves:
-            to_reassign_moves._action_assign()
+            self._handle_change_lot_reassign(
```

Comment:
Put in a separate method to avoid flake8 issue:
`stock_move_line_change_lot/models/stock_move_line.py:64:5: C901 'StockMoveLine.write' is too complex (17)`